packages/infrastructure/authentik_atlas_mongodb_sso/README.md

```diff
@@ -0,0 +1,84 @@
+# Atlas MongoDB SSO with Authentik
```

Comment:
The service is called MongoDB Atlas so the module should be named `authentik_mongodb_atlas_sso`